ztp/tools/siteconfig-converter/convert.go

```diff
@@ -965,3 +1002,85 @@ func writeClusterInstanceToFile(clusterInstance *ClusterInstance, filename strin
 
 	return nil
 }
+
+func generateKustomizationYAML(configMapName, configMapNamespace, manifestsDir, outputDir string) error {
+	fmt.Println("--- Kustomization.yaml Generator ---")
+
+	// Validate if the directory exists
+	if _, err := os.Stat(manifestsDir); os.IsNotExist(err) {
+		return fmt.Errorf("directory '%s' not found", manifestsDir)
+	}
+
+	// Get absolute path for debugging
+	absPath, err := filepath.Abs(manifestsDir)
+	if err != nil {
+		return fmt.Errorf("failed to get absolute path: %w", err)
+	}
+	fmt.Printf("Scanning directory: %s\n", absPath)
+
+	// Find all .yaml and .yml files in the specified directory (top-level only, no subdirectories)
+	var yamlFiles []string
+	entries, err := os.ReadDir(manifestsDir)
+	if err != nil {
+		return fmt.Errorf("failed to read directory: %w", err)
+	}
+
+	for _, entry := range entries {
+		if !entry.IsDir() {
+			ext := strings.ToLower(filepath.Ext(entry.Name()))
+			if ext == ".yaml" || ext == ".yml" {
+				// Get relative path from outputDir to the manifest file
+				relPath, err := filepath.Rel(outputDir, filepath.Join(manifestsDir, entry.Name()))
+				if err != nil {
+					return fmt.Errorf("failed to get relative path for %s: %w", entry.Name(), err)
+				}
+				yamlFiles = append(yamlFiles, relPath)
+				fmt.Printf("Found and adding: %s\n", relPath)
+			}
+		}
+	}
+
+	if len(yamlFiles) == 0 {
+		fmt.Printf("No .yaml or .yml files found in '%s'. Generating kustomization.yaml with empty files list.\n", manifestsDir)
+	}
+
+	// Sort files alphanumerically
+	sort.Strings(yamlFiles)
+
+	// Create the kustomization struct
+	kustomization := Kustomization{
+		ApiVersion: "kustomize.config.k8s.io/v1beta1",
+		Kind:       "Kustomization",
+		ConfigMapGenerator: []ConfigMapGenerator{
+			{
+				Files:     yamlFiles,
+				Name:      configMapName,
+				Namespace: configMapNamespace,
+			},
+		},
+		GeneratorOptions: GeneratorOptions{
+			DisableNameSuffixHash: true,
+		},
+	}
+
+	// Marshal to YAML
+	kustomizationData, err := yaml.Marshal(kustomization)
+	if err != nil {
+		return fmt.Errorf("failed to marshal kustomization to YAML: %w", err)
+	}
+
+	kustomizationContent := string(kustomizationData)
+
+	outputFile := filepath.Join(outputDir, KustomizationConfigMapGeneratorSnippetFile)
+	if err := os.WriteFile(outputFile, []byte(kustomizationContent), 0644); err != nil {
+		return fmt.Errorf("failed to write %s: %w", KustomizationConfigMapGeneratorSnippetFile, err)
+	}
```

Comment:
What happens if the tool is used to convert multiple SiteConfig files into the same output directory. Will this overwrite them all?  Or is this enforcing one converted site per output directory? 
The kbase article we have shows all the ClusterInstance CRs in the same directory (eg [here](https://access.redhat.com/articles/7124143#transitioning-sno2))